### PR TITLE
Added workaround for dialog flickering.

### PR DESCRIPTION
**Pull request by Reezaali**

### CODE REVIEW

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/debug-office-add-ins-on-ipad-and-mac.md](https://github.com/OfficeDev/office-js-docs-pr/blob/reeza-debug-mac/docs/testing/debug-office-add-ins-on-ipad-and-mac.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C2%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201811021758099487-539%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=tbJsj%2BAxWhEwcg4%2FwV7h9ITPqe2DTFPLRsgMx0XH0hg%3D&st=2018-11-02T17%3A54%3A24Z&se=2018-12-03T17%3A59%3A24Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/testing/debug-office-add-ins-on-ipad-and-mac.md [hunk 1]
```diff
@@ -28,6 +28,12 @@ Then, open the Office application and insert your add-in. Right-click the add-in
 
 > [!NOTE]
 > Please note that this is an experimental feature and there are no guarantees that we will preserve this functionality in future versions of Office applications.
+>
+> If you are trying to use the inspector, and the dialog is flickering, try the following workaround:
```
Comment (AlexJerabek) [inline docs/testing/debug-office-add-ins-on-ipad-and-mac.md]: Nit: I don't think the first comma is necessary.

File: docs/testing/debug-office-add-ins-on-ipad-and-mac.md [hunk 2]
```diff
@@ -28,6 +28,12 @@ Then, open the Office application and insert your add-in. Right-click the add-in
...
+> 1. Resize the dialog such that it's a very small window.
```
Comment (AlexJerabek) [inline docs/testing/debug-office-add-ins-on-ipad-and-mac.md]: Is "very small" descriptive enough? Do users need to make the window as small as possible? Or just smaller than the new window?

File: docs/testing/debug-office-add-ins-on-ipad-and-mac.md [hunk 3]
```diff
@@ -28,6 +28,12 @@ Then, open the Office application and insert your add-in. Right-click the add-in
...
+> 2. Choose **Inspect Element** which opens in a new window.
```
Comment (AlexJerabek) [inline docs/testing/debug-office-add-ins-on-ipad-and-mac.md]: Nit: `Choose **Inspect Element**` -> `Choose **Inspect Element**,`

File: docs/testing/debug-office-add-ins-on-ipad-and-mac.md (hunk 1 above)
Comment (kbrandl) [reply docs/testing/debug-office-add-ins-on-ipad-and-mac.md]: +1

File: docs/testing/debug-office-add-ins-on-ipad-and-mac.md (hunk 3 above)
Comment (kbrandl) [reply docs/testing/debug-office-add-ins-on-ipad-and-mac.md]: +1 (i.e., add a comma after `**Inspect Element**`)

File: docs/testing/debug-office-add-ins-on-ipad-and-mac.md (hunk 1 above)
Comment (Reezaali) [reply docs/testing/debug-office-add-ins-on-ipad-and-mac.md]: Done

File: docs/testing/debug-office-add-ins-on-ipad-and-mac.md (hunk 2 above)
Comment (Reezaali) [reply docs/testing/debug-office-add-ins-on-ipad-and-mac.md]: Updated

File: docs/testing/debug-office-add-ins-on-ipad-and-mac.md (hunk 3 above)
Comment (Reezaali) [reply docs/testing/debug-office-add-ins-on-ipad-and-mac.md]: Done

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/debug-office-add-ins-on-ipad-and-mac.md](https://github.com/OfficeDev/office-js-docs-pr/blob/reeza-debug-mac/docs/testing/debug-office-add-ins-on-ipad-and-mac.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C2%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201811022153150095-539%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=q7ajUflCxaaMVRrr8vd02GWNgGQfPzp%2FG7IaQBqP8GY%3D&st=2018-11-02T21%3A49%3A46Z&se=2018-12-03T21%3A54%3A46Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by Reezaali** [comment]
@kbrandl @AlexJerabek please review.

**Pull request merged**